### PR TITLE
Include '0' financials in submitted documents

### PR DESCRIPTION
**Pull request by markolson**

### CODE REVIEW

File: src/server/enrollment-system.js [hunk 1]
```diff
@@ -241,19 +241,19 @@ function veteranToSpouseInfo(veteran) {
  */
 function resourceToIncomeCollection(resource) {
   const incomeCollection = [];
-  if (resource.grossIncome) {
+  if (resource.grossIncome !== undefined) {
```
Comment (jbalboni) [inline src/server/enrollment-system.js]: This is going to allow nulls through, which might trip us up later. Maybe `!isNaN(resource.grossIncome)`?

**Review by jbalboni — APPROVED** [review]
Looks ok to me, though not sure about the `undefined` check.

File: src/server/enrollment-system.js (hunk 1 above)
Comment (markolson) [inline src/server/enrollment-system.js]: I may be misunderstanding this, but `!isNaN(null) == true`, so null would still get through with that.

Comment (jbalboni) [inline src/server/enrollment-system.js]: Ugh, that's true. In any case, I think we should have a check that checks for null and undefined, to be safe.

Comment (markolson) [inline src/server/enrollment-system.js]: lodash to the rescue.

**Pull request merged**